<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>365</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>244</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>244</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>244</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="button7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>100</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button8">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button9">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button6">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button5">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
      <horstretch>56</horstretch>
      <verstretch>60</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>243</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>243</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>243</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>304</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #dadbde, stop:1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
	background-color: rgb(215,215,215);
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #BEBEBE, stop:1 #D7D7D7)
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDiv">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
	color:white;
	background-color: rgb(255,151,57);
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832, stop:1 #FF9739)
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEqual">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>304</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
	color:white;
	background-color: rgb(255,151,57);
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832, stop:1 #FF9739)
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMinus">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>182</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
	color:white;
	background-color: rgb(255,151,57);
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832, stop:1 #FF9739)
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMult">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>121</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
	color:white;
	background-color: rgb(255,151,57);
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832, stop:1 #FF9739)
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlus">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>243</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
	color:white;
	background-color: rgb(255,151,57);
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832, stop:1 #FF9739)
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBack">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
	background-color: rgb(215,215,215);
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #BEBEBE, stop:1 #D7D7D7)
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPoint">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:1px solid grey;
	background-color: rgb(215,215,215);
}
QPushButton:pressed{
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #BEBEBE, stop:1 #D7D7D7)
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>244</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid grey;
	padding: 1px 1px 1px 1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
